src/pf-donut-chart/index.html

```diff
@@ -0,0 +1,167 @@
+
+<body>
+  <h3> Utilization chart with custom tooltip </h3>
+  <div id="chart"></div>
+  <h3> Relationship of a Set of Values to a Whole </h3>
+  <div id="chart1"></div>
+  <div id="chart2"></div>
+  <div id="chart3"></div>
+  <h3> Small Donut </h3>
+  <div id="chart4"></div>
+  <div style="text-align: center; width: 220px; height: 100px;">CSV Data</div>
+
+
+
+  <pf-donut-chart id="donut" title="Utilization Chart" target-selector="#chart" columns="[
+                                                                                  ['Used', 60],
+                                                                                  ['Available', 40]
+                                                                                ]" data="{
+                                                                                  'groups': [
+                                                                                      ['used', 'available']
+                                                                                  ],
+                                                                                  'order': null
+                                                                                }" width="200" height "171"></pf-donut-chart>
+
+  <pf-donut-chart id="donut1" title=" Animals donut1" target-selector="#chart1" rows="[
+                                                                        ['Dogs', 'Cats', 'Fish', 'Hamsters'],
+                                                                          [2, 2, 3, 1]
+                                                                          ]" width="200" height="171"></pf-donut-chart>
+
+  <pf-donut-chart id="donut2" target-selector="#chart2" legend="{
+                                              'show': true,
+                                              'position': 'right'
+                                            }" colors="{
+                                              'data1': '#0088ce',
+                                              'data2': '#3f9c35',
+                                              'data3': '#ec7a08' 
+                                            }" json="[
+                                              {'name': 'www.site1.com', 'data1': 200, 'data2': 200, 'data3': 400},
+                                              {'name': 'www.site2.com', 'data1': 100, 'data2': 300, 'data3': 400}
+                                            ]" data="{ 
+                                              'keys': {
+                                              'value': ['data1', 'data2', 'data3'] 
+                                              }
+                                            }" title="donut2" width="200" height="171"></pf-donut-chart>
+  <pf-donut-chart id="donut3" title="donut3" target-selector="#chart3" url="https://gist.githubusercontent.com/sahil143/b0c56761599c64080a3a042ad779e097/raw/97c51c6c4db4a121b7f49e7f58cca675c8528058/test.json"></pf-donut-chart>
```

Comment:
Is the plan to look at hosting this under the ```patternfly-webcomponents``` repository, or within a gist controlled by PatternFly?